samples/bluetooth/mesh/light/src/smp_dfu.c

```diff
@@ -11,54 +11,134 @@
 #include <zephyr/mgmt/mcumgr/smp_bt.h>
 
 #include <zephyr/bluetooth/bluetooth.h>
+#include <zephyr/bluetooth/conn.h>
+
+static void pending_adv_start(struct k_work *work);
 
 static struct bt_le_ext_adv *adv;
-static struct bt_le_ext_adv_start_param ext_adv_param = { .num_events = 0, .timeout = 1000 };
+static struct bt_le_ext_adv_start_param ext_adv_param = { .num_events = 0, .timeout = 0 };
```

Comment:
Can you add a note here? I noticed that this causes too many ads to get injected into surroundings when 3-4 devices are advertising SMP services.
```suggestion
/* This will result in the device advertising SMP service continuously. If this is undesirable, set timeout to a finite value, and then create a mechanism in the 
application (e.g. by pressing a button) to start SMP advertisements when needed by
calling @ref bt_le_ext_adv_start. */
static struct bt_le_ext_adv_start_param ext_adv_param = { .num_events = 0, .timeout = 0 };
```